main.go

```diff
@@ -89,33 +104,41 @@ func main() {
 
 	parseFlags()
 	testNetwork := network.New(
-		network.Nodes(config.NodesCount, multiverse.NewNode, network.ZIPFDistribution(config.ZipfParameter, float64(config.NodesTotalWeight))),
-		network.Delay(30*time.Millisecond, 250*time.Millisecond),
-		network.PacketLoss(0, 0.05),
+		network.Nodes(config.NodesCount, multiverse.NewNode, network.ZIPFDistribution(
+			config.ZipfParameter, float64(config.NodesTotalWeight))),
+		network.Delay(time.Duration(config.DecelerationFactor)*time.Duration(config.MinDelay)*time.Millisecond,
+			time.Duration(config.DecelerationFactor)*time.Duration(config.MaxDelay)*time.Millisecond),
+		network.PacketLoss(0, config.PayloadLoss),
 		network.Topology(network.WattsStrogatz(4, 1)),
 	)
 	testNetwork.Start()
 	defer testNetwork.Shutdown()
 
-	awResultsWriters := monitorNetworkState(testNetwork)
-	defer flushWriters(awResultsWriters)
+	resultsWriters := monitorNetworkState(testNetwork)
+	defer flushWriters(resultsWriters)
 	secureNetwork(testNetwork, config.DecelerationFactor)
 
-	time.Sleep(2 * time.Second)
+	// time.Sleep(2 * time.Second)
+
+	// To simulate the confirmation time w/o any double spendings, the colored msgs are not to be sent
+
+	// Here we simulate the double spending
```

Comment:
I am not sure if this will make any difference if we issue conflicts from one or two different peers. Because those nodes will process it as any other message and still will have its own opinion that will influence message processing. Only one diffence is that it will go to the same part of the network (the same neighbors) so the conflict will be resolved even faster.

Reply:
It's a good question. I think it depends on which phenomena we want to observe.
To rule out any uncertain behavior and to observe the cumulative approval weights of the double spent messages, we can first assume the double spent messages are issued by the same node.